File: dotnet/src/Pactman.NonprofitCheckPlus/Exceptions/PactmanConfigurationException.cs
namespace Pactman.NonprofitCheckPlus.Exceptions
{
    /// <summary>
    /// The client options were unusable — a missing API key, a malformed base URL.
    /// </summary>
    public sealed class PactmanConfigurationException : PactmanException
    {
        /// <summary>Initializes the exception.</summary>
        /// <param name="message">What was wrong with the options, and how to fix it.</param>
        public PactmanConfigurationException(string message)
            : base(message, ErrorCategory.Configuration, ErrorOrigin.Local)
        {
        }
    }
}

File: java/sdk/src/main/java/org/pactman/nonprofitcheckplus/exceptions/PactmanConfigurationException.java
package org.pactman.nonprofitcheckplus.exceptions;

/** The client options were unusable — a missing API key, a malformed base URL. */
public class PactmanConfigurationException extends PactmanException {

    private static final long serialVersionUID = 1L;

    /**
     * Creates the exception.
     *
     * @param message what was wrong with the options.
     */
    public PactmanConfigurationException(String message) {
        super(message, ErrorCategory.CONFIGURATION, ErrorOrigin.LOCAL, null);
    }
}
